src/cli.rs

```diff
@@ -60,11 +60,19 @@ pub struct CommandLineArgs {
     /// Default is 1 day.
     #[arg(long, default_value_t = 86400, env = "REDUCTIONIST_CHUNK_CACHE_AGE")]
     pub chunk_cache_age: u64,
+    /// Minimum interval in seconds between expiring chunks based on ttl.
```

Comment:
Does this need reworded or am I just misunderstanding?
```suggestion
    /// Minimum interval in seconds between checking for expired chunks based on ttl.
```